x-pack/plugins/alerting/server/alerting_authorization_client_factory.test.ts

```diff
@@ -60,7 +39,7 @@ test('creates an alerting authorization client with proper constructor arguments
     securityPluginStart,
     ...alertingAuthorizationClientFactoryParams,
   });
-  const request = CoreKibanaRequest.from(fakeRequest);
+  const request = mockRouter.createKibanaRequest();
```

Comment:
not necessary for this PR, but saw this failing when running Jest tests and decided to clean up by using core's mock instead.